Automattic/delphin
# Remove Docker configuration files

This pull request removes files that are unnecessary since we actually never used Docker.

#### Testing instructions

1. Run `git checkout remove/docker` and start your server, or open a [live branch](https://delphin.live/?branch=remove/docker)
2. Open the [`Home` page](http://delphin.localhost:1337/)
3. Check that everything works as before

#### Reviews

 
@Automattic/sdev-feed
